README.md

```diff
@@ -44,6 +45,36 @@ To run the tests
     make test
 
 
+.gitAttributes
+--------------
+
+Like in linguist you can override the strategies via `.gitattributes` file.
+Add a `.gitattributes` file to the directory and use the enry matchers `enry-documentation`,`enry-language` or `enry-vendored` to do the override.
```

Comment:
its only a name they called linguist-documentation or linguist-vendored because is part of linguist, I thought that because we are on enry we should call it enry-vendored and enry-documentation.